compiler/AST/ParamForLoop.cpp

```diff
@@ -532,6 +633,22 @@ void ParamForLoop::validateLoop(VarSymbol* lvar,
   }
 }
 
+void ParamForLoop::validateLoop(EnumSymbol* lvar,
+                                EnumSymbol* hvar,
+                                VarSymbol* svar) {
+  if (!lvar            || !hvar            || !svar)
```

Comment:
I known it's a copy of existing code, however especially in this case it is a bit unreadable. I suggest leaving a single space around `||` above.

Reply:
Done (for the original copy as well).